migration/FromMySqlToPostgreSql/FromMySqlToPostgreSql.php

```diff
@@ -712,7 +712,12 @@ private function arrangeColumnsData(array $arrColumns)
                 || stripos($arrColumn['Type'], 'linestring') !== false
                 || stripos($arrColumn['Type'], 'polygon') !== false
             ) {
-                $strRetVal .= 'HEX(ST_AsWKB(`' . $arrColumn['Field'] . '`)),';
+                if ( strcmp( '5.5', substr( $this->mysqlVersion, 0, 3 ) ) == 0
```

Comment:
AsBinary() and AsWKB() are deprecated as of MySQL 5.7.6 and will be removed in a future MySQL release. 

As far as I know ST_AsWKB will only work for 5.6 and above. 

Would you prefer to see:

if( mysql <= 5.5 )
{
      AsWKB
}
else
{
      ST_AsWKB
}


I will modify the PR how ever you see fit.